### PR TITLE
added cup of join

### PR DESCRIPTION
**Pull request by noa-seneor**
cup of join

### CODE REVIEW

File: cup_of_join.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def join(*parameters, sep='-'):
```
Comment (itayday) [inline cup_of_join.py:4]: Please add documantation to your function.

File: cup_of_join.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    l = []
+    if not parameters:
+        return None
+    for i in range(len(parameters)-1):
```
Comment (itayday) [inline cup_of_join.py]: Is there a way to simplify your code here? (what about using list comprehension?)

File: cup_of_join.py [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+
 def join(*parameters, sep='-'):
-    l = []
+    """
+    concatenate the lists received as parameters with
+    separator received as parameter or with "-"
+    :param parameters:
+    :param sep:
+    :return: concatenated list
+    """
```
Comment (lovodkin93) [inline cup_of_join.py:9]: please explain the purpose of the paramters in the documentation.

**Review by lovodkin93 — CHANGES_REQUESTED** [review]

File: cup_of_join.py (hunk 3 above)
Comment (lovodkin93) [reply cup_of_join.py:9]: much better!

**Review by lovodkin93 — APPROVED** [review]

**Pull request merged**